src/app/components/routing/application-routing.component.ts

```diff
@@ -38,9 +38,9 @@ export class ApplicationRoutingComponent implements OnInit {
       if (bookingFeatureToggle && AppUtils.isBookableAndJudicialRole(userDetails)) {
         return this.router.navigate([ApplicationRoutingComponent.bookingUrl]);
       }
-      if (userDetails && userDetails.userInfo && userDetails.userInfo.roles
-        && !userDetails.userInfo.roles.includes('pui-case-manager')) {
-        const userRoles = userDetails.userInfo.roles;
+      if (userDetails && userDetails.userInfo && userDetails.userInfo?.roles
```

Comment:
I'm going to be picky here. Should be 
if (userDetails?.userInfo?.roles && userDetails.roles.includes('pui-case-manager')
Don't need to test value and use optional chaining, and you don't need to use it for both operands of an &&
If you've checked it in an if, you don't need to use optional chaining inside the if block.

It doesn't matter functionally, but it's messy